README.md

```diff
@@ -120,6 +121,8 @@ end
 
 Open up the `hashketball.rb` file and add the line *require "pry"* at the top and copy and paste the above method. Then, beneath the `end` that closes the method definition, call the method (`good_practices`) and, in your terminal, run the file with `ruby hashketball.rb`. Play around with the methods in each binding until you get comfortable with the iteration. This should give you a stronger sense of how we iterate through so many levels of a nested hash and what happens on each level. **Use this method of placing LOTS of bindings when you iterate in order to solve this lab.**
 
+*If you are getting errors when attempting to use pry, make sure you call* `gem install pry` in your terminal.
+
 Okay, *now* we're ready to build out methods:
 
 ### Method Building
```

Comment:
Should we add name into the expected response for player_stats on 144? This would need to be updated in the tests as well.

Reply:
yes that sounds good––i'll update

Reply:
this particular method should take in a player name and return all of their data _except_ for their name

Reply:
we could change it as i'm not sure asking students to remove a key/value pair from a hash is particularly useful

Reply:
not sure we are on the right track here -- see comment on pull request. separately:

a method called `player_stats` that expects a name and returns stats makes sense from my perspective. the name is already known (its whats passed), its the stats we want. This change is in response to the hash --> array change, which is addressed in comment on pr.